src/test/kotlin/CheckersModelTest.kt

```diff
@@ -0,0 +1,40 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertTrue
+
+class TestCanMove {
```

Comment:
Good test, but try to write more in the future. And keep the rule in mind: if you write some code, you should write the tests checking it.